applications/create-substrate-app.md

```diff
@@ -0,0 +1,120 @@
+
+https://github.com/philoniare/
+
+### Team LinkedIn Profiles (if available)
+
+- https://linkedin.com/in/tuguldurb
+
+
+## Development Status
+
+There's already an [existing template](https://github.com/substrate-developer-hub/substrate-front-end-template). We will be building on top of this template and adding more features to it.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 1 month
+- **Full-Time Equivalent (FTE):**  1 FTE
+- **Total Costs:** 5,000 USD
+
+### Milestone 1 - Full CLI functionality
+
+- **Estimated duration:** 1 month
+- **FTE:**  1
+- **Costs:** 5,000 USD
+
+| Number | Deliverable                           | Specification                                                                                                                                                   |
+| -----: |---------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **0a.** | License                               | Apache 2.0                                                                                                                                                      |
+| **0b.** | Documentation                         | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) call the CLI                      |
+| **0c.** | Testing and Testing Guide             | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0e. | Article                               | We will publish an **article**/workshop that explains how to run the CLI (what was done/achieved as part of the grant).                                         |
+| 1. | TypeScript React Front-End Template   | Create a modified version of React template with TypeScript                                                                                                     |
+| 2. | JavaScript Vue Front-End Template     | Create a JavaScript version of Vue Front-End Template                                                                                                           |
+| 3. | TypeScript Vue Front-End Template     | Create a TypeScript version of Vue Front-End Template                                                                                                           |
+| 4. | JavaScript Angular Front-End Template | Create a JavaScript version of Angular Front-End Template                                                                                                       |
+| 5. | TypeScript Angular Front-End Template | Create a TypeScript version of Vue Front-End Template                                                                                                           |
```

Comment:
Personally, I'd be in favor of supporting TypeScript-only. I don't see any major advantage in that, and I don't see any potential benefit in supporting JavaScript implementations for these frameworks.

Reply:
Yeah, that would be the case for people experienced in TypeScript, but for new devs just getting started, unfamiliar with TypeScript, it would present additional barrier for entry. As a general tool, we want to support both versions. We can definitely make the TypeScript version the default option when they choose though.

Reply:
Agreed, developers using projects setup with TypeScript can still write in JavaScript if they choose to do so.